GliaWidgets/Sources/CoreSDKClient/CoreSDKClient.Mock.swift

```diff
@@ -367,4 +367,96 @@ extension CoreSdkClient {
     }
 }
 
+extension CoreSdkClient.QueueState {
```

Comment:
I think this will not be needed anymore because now the Queue init will be public.